core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -677,7 +677,7 @@ class ZkUtils(val zkClient: ZkClient,
     topics
   }
 
-  def getPartitionReassignmentZkData(partitionsToBeReassigned: Map[TopicAndPartition, Seq[Int]]): String = {
+  def formatAsReassignmentJson(partitionsToBeReassigned: Map[TopicAndPartition, Seq[Int]]): String = {
```

Comment:
Having had another look it seems pretty debatable whether extracting this out further really makes things better so I'm inclined to leave it as is. If you feel strongly I'll change.